Profiles/ProfilesAPI.py

```diff
@@ -1,6 +1,64 @@
-from flask import Blueprint
+from flask import Blueprint, make_response, request
 import sys
 from db import db
+from flask_cors import cross_origin
+
 sys.path.append("../")
 
-profiles_api = Blueprint("profiles", __name__)
\ No newline at end of file
+profiles_api = Blueprint("profiles", __name__)
+
+
+@profiles_api.route("/<int:id>", methods=["GET"])
+@cross_origin(supports_credentials=True)
+def get_profile(id):
+    try:
+        data = {'name': db[id]['name'], 'scores': db[id]['scores']}
+        response = {'status': 'success', 'data': data}
+    except Exception as e:
+        return {"err": "An error has occured", "status": " failed"}, 500
+
+    return make_response(response)
+
+
+@profiles_api.route("/profiles", methods=["POST"])
+@cross_origin(supports_credentials=True)
+def create_new_profile():
+    try:
+        data = request.get_json()
+        db.append(data)
+        response = {'status': 'success'}
+    except Exception as e:
+        return {"err": "An error has occured", "status": " failed"}, 500
+
+    return make_response(response)
+
+
+@profiles_api.route("/<int:id>", methods=["DELETE"])
+@cross_origin(supports_credentials=True)
+def delete_profile(id):
+    try:
+        id = int(id)
+        deleted = db[id]
+        new_db = [profile for profile in db if profile != deleted]
```

Comment:
consider del db[id] since doing this does not "delete" them from db variable. 
e.g. DELETE request to delete id 0. output tells me that id 0 has been removed and the new_db does not have id 0. then I proceed to DELETE request to delete id 1. output tells me that id 1 has been removed but the new_db still have that id 0 person who is not supposed to be there.